src/services/users/userTypes.ts

```diff
@@ -0,0 +1,19 @@
+export interface createUserRequest {
```

Comment:
This is a copy of the one in User Request types, we should try to re-use this type or something instead

Reply:
Yeah we should use `extends` here @nie209

Reply:
Also class name should use upper pascal case

Reply:
ohhh this file i forgot to remove it. all the types is in a different place. we have a type folder  within the folder that contain two file `userRequestTypes.ts` and `useResponseTypes.ts`